lib/clients/http_client/http_client.ts

```diff
@@ -141,13 +142,28 @@ export function createHttpClientClass(
       const url = `${scheme}://${this.domain}${this.getRequestPath(
         params.path,
       )}${ProcessedQuery.stringify(params.query)}`;
+
       const request: NormalizedRequest = {
         method: params.method,
         url,
         headers: canonicalizeHeaders(headers as any),
         body,
       };
 
+      if (config.logQueries) {
+        const message = [
+          'Making HTTP request',
+          `${request.method} ${request.url}`,
+          `Headers: ${JSON.stringify(headers)}`,
+        ];
+
+        if (body) {
+          message.push(`Body: ${JSON.stringify(body).replace(/\\n/g, '\n')}`);
+        }
+
+        debug(config, message.join('  -  '), {domain: this.domain});
```

Comment:
I think `debug`  here makes it confusing that this is a log. so I would import this as `log`/`logger` and then this would be `log.debug`/`logger.debug`